### PR TITLE
PRM Automated Test | LPS-172427 | master

### PR DESCRIPTION
**Pull request by mateussandes23**
**Jira ticket:** https://issues.liferay.com/browse/LPS-172427

### CODE REVIEW

**Comment by mateussandes23** [comment]
ci:test:sf

**Comment by mateussandes23** [comment]
ci:test:sf

**Comment by mateussandes23** [comment]
ci:test:sf

**Comment by mateussandes23** [comment]
ci:test:sf

**Comment by mateussandes23** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro [hunk 1]
```diff
@@ -147,6 +147,51 @@ definition {
...
+				key_fieldName = "activities[${activityIndex}].budgets[${budgetIndex}].cost",
+				locator1 = "PRMMDFRequest#TEXT_FIELD",
+				value1 = ${bugdetValue});
+
+			var budgetIndex = ${budgetIndex} + 1;
+		}
+
+	}
+
+	@summary = "verify if the requested amount is equal to the mdf values"
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro]: Hi @mateussandes23, could you please rename this summary to 'Verify if the MDF Requested amount is equal to 50% of the total value'?

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro [hunk 2]
```diff
@@ -147,6 +147,51 @@ definition {
...
+	macro verifyRequestedAmount {
+		if (!(isSet(activityIndex))) {
+			var activityIndex = 0;
+		}
+
+		AssertTextEquals(
+			key_fieldName = "activities[${activityIndex}].mdfRequestAmount",
+			locator1 = "PRMMDFRequest#TEXT_FIELD",
+			value1 = ${discountedValue});
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro]: Hi @mateussandes23, this macro should calculate the Total MDF Requested Amount. The calculation must be done by dividing the budget value by 2.

After this, you should assert that the value on the field is equals to your result.

**Comment by AfonsoHSantos** [comment]
Hi @mateussandes23, could you please rename [this commit](https://github.com/pt-liferay-solutions/liferay-portal/pull/3379/commits/aa998454dc02454571ad717705397154cfbd0834) to match with the commit content? Thanks!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 3]
```diff
@@ -488,6 +482,39 @@ definition {
 		}
 	}
 
+	@description = "Verify that the value of Total MDF Requested Amount* will be according to the Total cost and Claim percent"
+	@priority = 5
+	test RequestedValueIsEqualsTo50PercentOfTotalCost {
+		task ("Given a new MDF request") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:489]: Hi @mateussandes23, could you please rename this task to 'Given that I fill the MDF until Activity page'

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 4]
```diff
@@ -488,6 +482,39 @@ definition {
...
+				campaignName = "Campaing Name",
+				countryName = "Mexico",
+				goalsOptions = "Lead generation",
+				targetMarketOptions = "Education,Energy",
+				targetRolesOptions = "Administrator");
+
+			Button.click(button = "Continue");
+		}
+
+		task ("When I create a new activity") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:505]: Hi @mateussandes23, could you please rename this task to 'When I go to add a new activity'

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 5]
```diff
@@ -488,6 +482,39 @@ definition {
...
+			PRMMDFRequest.addNewActivity();
+		}
+
+		task ("And When I add a new expense") {
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase:509]: Hi @mateussandes23, could you please rename this task to 'And I add a new expense'

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 6]
```diff
@@ -488,6 +482,39 @@ definition {
...
+			PRMMDFRequest.fillRequestAmountField(
+				budgetList = "Direct Mail-100000,Room Rental-3000");
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase]: Hi @mateussandes23, here isn't necessary to add two budgets. Could you please add just one? Thanks!

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase [hunk 7]
```diff
@@ -488,6 +482,39 @@ definition {
...
+		}
+
+		task ("Then the requested amount is equal to 50% of the MDF value"){
```
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFRequest.testcase]: Hi @mateussandes23, could you please rename this task to 'Then the Total MDF Requested Amount is equals to 50% of the MDF value'

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro (hunk 2 above)
Comment (AfonsoHSantos) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/macros/PRMMDFRequest.macro]: Hi @mateussandes23, could you please rename this variable to `requestedAmount`?

**Review by AfonsoHSantos — CHANGES_REQUESTED** [review]

**Pull request closed without merging**